acra/configuring-maintaining/tls/_index.md

```diff
@@ -2,3 +2,146 @@
 title: TLS
 bookCollapseSection: true
 ---
+
+# TLS
+
+## Common TLS flags
+
+(common for `acra-connector`, `acra-server`, `acra-translator`)
+
+* `--tls_auth=<mode>`
+
+  Set authentication mode that will be used in TLS connection with AcraConnector and database.
+
+  * `0` — do not request client certificate, ignore it if received
+  * `1` — request client certificate, but don't require it
+  * `2` — expect to receive at least one certificate to continue the handshake
+  * `3` — don't require client certificate, but validate it if client actually sent it
+  * `4` — (default) request and validate client certificate
+
+  These values correspond to [crypto.tls.ClientAuthType](https://golang.org/pkg/crypto/tls/#ClientAuthType).
+
+* `--tls_key=<filename>`
+
+  Path to private key that will be used in AcraServer's TLS handshake
+  with AcraConnector as server's key and database as client's key.
```

Comment:
`--tls_key` flag is presented for Server/Connector/Translator, right? Then it is not clear from this phrase how this key is used. I assume Acra service (Server/Connector/Translator) acts as a server and database as a client, is it?